easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.5.5-foss-2016a-mpi.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'CMakeMake'
+name = 'IQ-TREE'
+version = '1.5.5'
+versionsuffix = "-mpi"
+
+homepage = 'http://www.iqtree.org/'
+description = """Efficient phylogenomic software by maximum likelihood"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+github_account = 'Cibiv'
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+
+configopts = '-DIQTREE_FLAGS="mpi omp" -DCMAKE_C_COMPILER=mpicc -DCMAKE_CXX_COMPILER=mpicxx'
+
+builddependencies = [('CMake', '3.5.2')]
+
+sanity_check_paths = {
+    'files': ["bin/iqtree-omp-mpi"],
+    'dirs': ["bin"],
```

Comment:
no need to check for a non-empty `bin` if you're already checking for a binary file in that subdirectory